server/src/main/java/org/candlepin/resource/DistributorVersionResource.java

```diff
@@ -139,26 +111,16 @@ else if (!StringUtils.isBlank(capability)) {
             this.translator.getStreamMapper(DistributorVersion.class, DistributorVersionDTO.class));
     }
 
-    @ApiOperation(notes = "Deletes a Distributor Version", value = "delete")
-    @ApiResponses({ @ApiResponse(code = 400, message = ""), @ApiResponse(code = 404, message = "") })
-    @DELETE
-    @Produces(MediaType.WILDCARD)
-    @Path("/{id}")
-    public void delete(@PathParam("id") String id,
-        @Context Principal principal) {
```

Comment:
@Januson What was to the purpose of `@Context Principal principal` in this API?

Reply:
Principal is unused so it should be save to remove. Just including it should have any side effects.